crates/oxc_linter/src/snapshots/adjacent_overload_signatures.snap

```diff
@@ -0,0 +1,263 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: adjacent_overload_signatures
+---
+  ⚠ typescript-eslint(adjacent-overload-signatures): All "foo" signatures should be adjacent.
+   ╭─[adjacent_overload_signatures.tsx:4:1]
+ 4 │         type bar = number;
+ 5 │         function foo(sn: string | number) {}
```

Comment:
Is it possible to pick the closest two labels that are not adjacent so we can only have once diagnostic per group? 🤔